### PR TITLE
Add functional requirements description in gherkin

### PR DESCRIPTION
**Pull request by Gochad**
## Changes:
1. Description about what we expect from app
2. Test model and parser argument for exec app with test model and without it
## Jira ticket
https://jira.frege.ii.uj.edu.pl/browse/GOC-2

### CODE REVIEW

File: apptests/feature/generators.feature [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    Then return filtered data
+
+  Scenario: We want to add additional info about videos
+    Given videos 
+    When we collect ids of videos 
+    And make request for info
+    Then we extend video info by duration 
+    And we extend video info by audio language
+
+  Scenario: Make transcript for every video in data
```
Comment (Kowalski1024) [inline apptests/feature/generators.feature]: I think, there should be more like "Returns information about available transcriptions".  
The [add_transcripts_info](https://github.com/Kowalski1024/Mi-Go/blob/master/generators/youtube_generator.py#L181C5-L181C25) returns only a list of languages in which transcriptions are available, divided into those added manually and those generated by YT. It's also response from api, so we don't transcibe video.

File: testrunners/tests/tests.py [hunk 2]
```diff
@@ -9,20 +9,24 @@
 import databases
 from libs.differs import jiwer_differ
 from testrunners.tests import TranscriptTest
-
+from apptests import TestModel
 
 class TranscriptDifference(TranscriptTest):
```
Comment (Kowalski1024) [inline testrunners/tests/tests.py]: Please adjust to the latest changes. I have added a new class [DummyTest](https://github.com/Kowalski1024/Mi-Go/blob/master/testrunners/tests/tests.py#L14) that should allow you to test the framework itself without loading the model.

**Review by Kowalski1024 — CHANGES_REQUESTED** [review]

File: apptests/feature/generators.feature (hunk 1 above)
Comment (Gochad) [reply apptests/feature/generators.feature]: @Kowalski1024 i changed that

File: testrunners/tests/tests.py (hunk 2 above)
Comment (Gochad) [reply testrunners/tests/tests.py]: fix after rebase

File: testrunners/tests/tests.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 import databases
 from libs.differs import jiwer_differ
 from testrunners.tests import TranscriptTest
-
+from apptests import TestModel
```
Comment (Kowalski1024) [inline testrunners/tests/tests.py]: The changes from the tests.py file should be rolled back, as DummyTest can be used.

File: apptests/feature/all_project.feature [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    When generate testplan
+    Then we get requested videos
+
+  Scenario: TestRunner must execute tests on a testplan, prepare the data for testing, run the tests, and store the results in a database
+    Given testplan
+    When register tests and trigger it
+    Then results store in database and JSON file
+  
+  Scenario: TranscriptTest runs model and compares its output to the yt transcript
+    Given model type
```
Comment (Kowalski1024) [inline apptests/feature/all_project.feature]: I would make it more general, like:
`Given arguments for a particular TranscriptTest`

Without model typle, language, etc.

Comment (Gochad) [reply apptests/feature/all_project.feature]: done

**Review by Kowalski1024 — APPROVED** [review]

**Pull request merged**